docs/source/guides/network_node/staking_guide.rst

```diff
@@ -302,25 +311,27 @@ to indicate which are the automated rules (in our case, allowing listing of acco
 
 .. code:: bash
 
-    $ clef --keystore /path/to/keystore --chainid 5 --advanced --rules rules.js
+    $ clef --keystore <PATH TO KEYSTORE> --chainid <CHAIN ID> --advanced --rules rules.js
 
 
 
 Initialize a new stakeholder
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Before continuing with stake initiation and management, A setup step is required to configure nucypher for staking.
-This will create a configuration file (`~/.local/share/nucypher/stakeholder.josn`) containing editable configuration values.
+This will create a configuration file (`~/.local/share/nucypher/stakeholder.json`) containing editable configuration values.
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake init-stakeholder --provider <PROVIDER> --network <NETWORK_NAME>
+    (nucypher)$ nucypher stake init-stakeholder --signer <SIGNER URI> --provider <PROVIDER> --network <NETWORK_NAME>
+
+where:
 
-If you ran ``geth`` node as above, your ``<PROVIDER>`` is
-``ipc:///home/<username>/.ethereum/goerli/geth.ipc``
-(on MacOS, ``ipc:///Users/<username>/Library/Ethereum/goerli/geth.ipc``)
+    * If you utilized `Clef Setup`_, the ``SIGNER URI`` is ``clef:///home/<username>/.clef/clef.ipc``
+    * If you ran ``geth`` node as above, your ``<PROVIDER>`` is ``ipc:///home/<username>/.ethereum/geth.ipc``
+      (on MacOS for Goerli testnet, ``ipc:///Users/<username>/Library/Ethereum/goerli/geth.ipc``)
```

Comment:
stray Goerli mention

Reply:
This was my attempt at having an addendum that mentioned if the person was developing on our testnet. Is your thought to remove "on MacOS for ~~Goerli~~ testnet" and just use `testnet`? If so, then what about the ipc provider uri that contains `goerli` as well?

I specifically mention testnet because MacOS shouldn't be used for mainnet 😅 .

Alternatively, I could just remove the `on MacOS` addendum, but we may get questions from developers using Macs and developing on testnet.

It's a weird balance.

Reply:
I see. I don't really have strong feelings either way then.

Reply:
Yeah - this supports my below point about needing to break this out into a separate guide altogether in a follow-up.